adam/semantics.py

```diff
@@ -135,6 +138,9 @@ class ObjectSemanticNode(SemanticNode):
         default=None, validator=optional(instance_of(str))
     )
 
+    def dot_label(self) -> str:
+        return " ".join(self.concept.debug_string)
```

Comment:
Yes, the debug string is type defined as a string, so that would be the resulting behavior. This was one of the functions which was previously defined in the elif ladder, and I wasn't really sure as to why it was set up this way before